resources/views/campaigns/edit.blade.php

```diff
@@ -24,6 +24,7 @@
                         >
                     </div>
 
+                @if (Auth::user()->role === 'admin')
```

Comment:
Would it be worth using the `isAdmin` helper for this? (I'm biased because I updated those helpers recently 🙃)
https://github.com/DoSomething/rogue/blob/261c3a67d16be6e9e982352cc3aa70c0e262e2a8/app/helpers.php#L94-L107

Reply:
LOL, totally I think the advantage here is that the helper keeps the logic to determine an admin encapsulated to one central location.

Reply:
oh I forgot we had that!!!!! will update to use it